woocommerce/wc-admin
# Navigation: Applying styles & behavior for folded menu, including mobile

Fixes #5326 

Updating styles and behaviour governing new navigation feature, including mobile. This should align with the following acceptance criteria: 

- A closed menu will appear as a square with the site logo of size equal to the wc-admin header.
- With the navigation closed, wc-admin or embedded pages occupy the entire screen width.
- On screens `$break-large: 960px` or below toggling the navigation pushes or overlaps the content, whichever is better.
- At the same breakpoint as above, the Navigation is collapsed by default.

I've also added a listener on the `orientationchange` event to handle the case of the layout for a device switching from landscape to portrait, in which case it will hide the navigation. This change is probably debatable usability wise, so I'm glad to remove it if desired.

### Screenshots
**Folded Before:** 
<img width="459" alt="Screen Shot 2020-10-23 at 3 36 04 PM" src="https://user-images.githubusercontent.com/1922453/96949927-91531500-1545-11eb-8743-66b72cde2998.png">

**Folded After:**
![image](https://user-images.githubusercontent.com/444632/97493543-57598880-1922-11eb-947f-66dd059b989f.png)

**Open on Desktop:**
![image](https://user-images.githubusercontent.com/444632/97493655-866ffa00-1922-11eb-86c3-fb52f5a11708.png)

**Open on Mobile:**
![image](https://user-images.githubusercontent.com/444632/97493734-9f78ab00-1922-11eb-8a24-8919001b83ac.png)


### Detailed test instructions:

- Navigate to WooCommerce pages with navigation feature on.
- Click logo in top-left to fold navigation.
- Ensure your screen width is greater than 960px.
- Observe that it now appears as the "Folded After" screenshot, and does **not** overlap content.
- Shrink viewport to less than 960px.
- With the navigation open, it should appear the same but now overlay the content.
- Refresh the page, and ensure that the  navigation collapses by default (while still with a small viewport).
- You can use Chrome or Firefox dev tools to test a `orientationchange` event to ensure that the navigation collapses when switching to a view smaller than `960px`.
